app/src/main/java/com/blinky/learnletters/SightSoundActivity.kt

```diff
@@ -1,14 +1,29 @@
 package com.blinky.learnletters
 
+import android.media.MediaPlayer
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import android.view.View
+import kotlinx.android.synthetic.main.sight_sound_layout.*
 
 class SightSoundActivity : AppCompatActivity() {
 
-    Button playButton = (Button) this.findViewById(R.id.playBtn)
+    private var mediaPlayer: MediaPlayer? = null
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.sight_sound_layout)
+
+        mediaPlayer = MediaPlayer.create(this, R.raw.letter_a)
+        mediaPlayer?.setVolume(1f, 1f)
+        mediaPlayer?.setOnPreparedListener {
+            println("READY TO GO")
+        }
```

Comment:
Create media player for sound "a"
Set volume to 100%
print logging to console (change to debug later)